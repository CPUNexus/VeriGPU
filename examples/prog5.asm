# test li with different registers, using outr
# (also test outr)

li x1 0x01
outr x1
li x1 0x02
outr x1
li x1 0x1e
outr x1
li x1 0x2d
outr x1
li x21 22
li x23 33
outr x21
outr x23
halt
