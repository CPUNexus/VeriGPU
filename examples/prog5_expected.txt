out 0 00000000000000000000000000000001 00000001 1
out 1 00000000000000000000000000000010 00000002 2
out 2 00000000000000000000000000011110 0000001e 30
out 3 00000000000000000000000000101101 0000002d 45
out 4 00000000000000000000000000010110 00000016 22
out 5 00000000000000000000000000100001 00000021 33